Source/CMakeLists.txt

```diff
@@ -48,7 +48,17 @@ set(SRCS ${DolphinProcessSrc}
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt6Widgets REQUIRED)
+if (WIN32)
+  find_package(Qt6Widgets QUIET)
+	if (NOT Qt6Widgets_FOUND)
+		message(STATUS "Qt package not found, using external lib")
+		list(APPEND CMAKE_PREFIX_PATH "${CMAKE_CURRENT_LIST_DIR}\\..\\Externals\\Qt\\Qt6.5.3\\x64")
```

Comment:
I see CI built successfully, but isn't CMAKE_CURRENT_LIST_DIR going to resolve to the level where "\\..\\" isn't necessary?

Reply:
The CI sets the `CMAKE_PREFIX_PATH` in the command line, so this code path is not being exercised in the CI right now.

Reply:
> I see CI built successfully, but isn't CMAKE_CURRENT_LIST_DIR going to resolve to the level where "\..\" isn't necessary?

I would have thought it'd resolve to the `Source/` directory, therefore `\..\` would be necessary.

Reply:
> > I see CI built successfully, but isn't CMAKE_CURRENT_LIST_DIR going to resolve to the level where "\..\" isn't necessary?
> 
> I would have thought it'd resolve to the `Source/` directory, therefore `\..\` would be necessary.

This may be correct, I didn't test it myself. I just wanted to ensure it is verified before merge.

Reply:
Verified it is indeed correct as is.